package/Shaders/Common/DeferredShared.hlsli

```diff
@@ -4,7 +4,7 @@ cbuffer PerFrameDeferredShared : register(b11)
 	float4 CameraData;
 	row_major float3x4 DirectionalAmbient;
 	uint FrameCount;
-	uint pad0[3];
+	uint FrameCountAlwaysActive;
```

Comment:
Properly padded? had 3 uints before, now only 1? Will this work on nvidia 1* cards etc?

Have a `uint pad0[2];` still in cpp side

Reply:
This should be fine